lib/features/debug/presentation/screens/debug/debug_screen.dart

```diff
@@ -31,10 +31,10 @@ class DebugScreen extends ElementaryWidget<IDebugScreenWidgetModel> {
 }
 
 class _Body extends StatelessWidget {
-  final ValueListenable<UrlType> urlState;
+  final ValueListenable<Url> urlState;
   final ValueListenable<ThemeMode> themeState;
-  final void Function(UrlType?) urlChanged;
-  final void Function(UrlType) switchServer;
+  final void Function(Url?) urlChanged;
```

Comment:
you can use `ValueChanged<Url?>` here

Reply:
Debug feature will be refactored in the separate pr.